src/llv8.h

```diff
@@ -7,6 +7,14 @@
 
 #include "src/llv8-constants.h"
 
+using lldb::ByteOrder;
+
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+#define HOST_BYTE_ORDER ByteOrder::eByteOrderLittle
+#else
+#define HOST_BYTE_ORDER ByteOrder::eByteOrderBig
+#endif
```

Comment:
I still think you should do this at runtime.  Both node and lldb do it that way.